### PR TITLE
chore: add myself to CODEOWNERS

### PR DESCRIPTION
**Pull request by henryiii**
This will ensure I get notified about pull requests where these files change.

### CODE REVIEW

**Comment by henryiii** [comment]
This would automate what @rwgk had to do manually for #2938.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+*.cmake @henryiii
+CMakeLists.txt @henryiii
+*.yml @henryiii
+*.yaml @henryiii
+/tools/ @henryiii
+/pybind11/ @henryii
```
Comment (rwgk) [inline .github/CODEOWNERS:6]: Did one `i` get cut off at the end?

**Review by rwgk — APPROVED** [review]
LGTM

File: .github/CODEOWNERS (hunk 1 above)
Comment (henryiii) [reply .github/CODEOWNERS:6]:
```suggestion
/pybind11/ @henryiiI
```

Comment (henryiii) [reply .github/CODEOWNERS]: Yes, thanks!

**Pull request merged**